LoginPlugin/iOS/README.md

```diff
@@ -0,0 +1,147 @@
+            }
+        } else {
+            completion(.failed)
+        }
+    }
+    
+    /**
+     This method is being called in order to start logout process.
+     The completion should always be called when the process is done - no matter what is the result.
+    */
+    public func logout(_ completion: @escaping ((ZPLoginOperationStatus) -> Void)) {
+        if let loginManager = loginManager {
+            loginManager.logOut()
+            completion(.completedSuccessfully)
+        } else {
+            completion(.failed)
+        }
+    }
+    
+    /**
+     This methood is called in order to verify if we need to start a login flow
+     Returns bool value indicating if the user is already verified if not  we start the login proccess
+     in this example we check if the login token exists and if it is valid
+    */
+    public func isAuthenticated() -> Bool {
+        var retVal = false
+        if let expirationDate = self.getUserExpirationDate(),
+            self.getUserToken().isEmpty == false {
+            let currentDate = Date()
+            if expirationDate > currentDate || expirationDate == currentDate {
+                retVal = true
+            }
+        }
+        return retVal
+    }
+    
+	/**
+     This methood is called in order to verify if authorization flow is in process
+    */
+    public func isPerformingAuthorizationFlow() -> Bool {
+        return self.isPerformingAuthorizationFlow()
+    }
+    ```
+
+### ZPLoginProviderUserDataProtocol
+``` swift  
+     /**
+     Getter to the user Token usually can be used for authentication check
+    */
+    public func getUserToken() -> String {
+        var token = ""
+        if let userToken = UserDefaults.standard.string(forKey: "fb_login_token") {
+            token = userToken
+        }
+        return token
+    }
+    
+    /**
+     Setter for the user Token - usually set after login success
+     @Params: the authentication recieved when login successfuly
+     */
+    public func setUserToken(token: String?) {
+        UserDefaults.standard.set(token, forKey: "fb_login_token")
+    }
+    
+    /**
+     This methood is called in order to verify if we need to start a login flow with respect to the policies dictionary
+     @params policies - dictionary containing policies to be considered when returning the result 
+     Returns bool value indicating if the user is already verified if not  we start the login proccess
+     in this example we check if the login token exists and if it is valid
+    */
+    public func isUserComply(policies:[String: NSObject]) -> Bool {
+        return self.isAuthenticated()
+    }
+```
+
+#### Using the Zapp Plugin Configuration JSON
+When creating a plugin in Zapp we can create custom configuration fields. This enable the Zapp user to fill relevant details for the specific plugin. More details can found in the [Zapp Plugin Manifest Format](http://zapp-tech-book.herokuapp.com/zappifest/plugins-manifest-format.html).
```

Comment:
change the url to 'http://developer-zapp.applicaster.com/zappifest/plugins-manifest-format.html'